Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -24,6 +26,23 @@
                          'so our content wizard @{selected_reviewer} will know the proposed changes are ready to be ' \
                          'reviewed.'
 
+XSOAR_SUPPORT_LEVEL_LABEL = 'Xsoar Support Level'
+PARTNER_SUPPORT_LEVEL_LABEL = 'Partner Support Level'
+COMMUNITY_SUPPORT_LEVEL_LABEL = 'Community Support Level'
+CONTRIBUTION_LABEL = 'Contribution'
+
+
+def parse_changed_files_names() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="Parse the changed files names.")
+    parser.add_argument(
+        "-c",
+        "--changed_files",
```

Comment:
I think it would be better to set a delimiter such as ";" instead of using the default ' ' as a separator for the file names. I think we had a problem with the default delimiter in the past  (in the doc reviewer workflow) and we solved it here - [pull/16719](https://github.com/demisto/content/pull/16719).